cypress.json

```diff
@@ -17,5 +17,6 @@
   "watchForFileChanges":false,
   "projectId": "ebzwp1",
   "pageLoadTimeout": 200000,
-  "defaultCommandTimeout": 30000
+  "defaultCommandTimeout": 30000,
+  "experimentalStudio" : true
```

Comment:
Have you noticed if the speed on CI is impacted by this? @the-greenman

Reply:
I didn't see any change. It almost looked faster.